src/main/java/org/folio/service/transactions/TransferService.java

```diff
@@ -56,25 +52,14 @@ public Future<Transaction> createTransaction(Transaction transfer, RequestContex
         transfer.getId()))
       .onFailure(e -> logger.error("createTransaction:: Transfer with id {} or associated data failed to be processed",
         transfer.getId(), e))
-      .map(v -> transfer)
-    );
+      .map(v -> transfer);
```

Comment:
not relates to this PR, but you can update in scope of it in methods updateBudgetsTransferFrom, updateBudgetsTransferTo.
 .map(i -> null) -> .mapEmpty()

Reply:
Actually I think we should use the created transfer instead of the initial one, and return it too. I am changing that code accordingly.